src/esp/physics/bullet/BulletRigidStage.cpp

```diff
@@ -149,7 +149,7 @@ void BulletRigidStage::constructBulletSceneFromMeshes(
                     btVector3{transformFromLocalToWorld.translation()}};
     std::unique_ptr<btRigidBody> sceneCollisionObject =
         std::make_unique<btRigidBody>(cInfo);
-    ASSERT(sceneCollisionObject->isStaticObject());
+    ASSERT(sceneCollisionObject->isStaticObject(), );
```

Comment:
I wonder if it wouldn't make sense to directly switch to `CORRADE_INTERNAL_ASSERT()`, since you're changing most (all?) occurences anyway :)

Reply:
I'd prefer that actually.  I'll do that, thanks for the reminder!